package.json

```diff
@@ -79,6 +79,7 @@
   "dependencies": {
     "bignumber.js": "^9.1.1",
     "ethereumjs-util": "^7.1.5",
+    "ethers": "^5.7.2",
```

Comment:
Why? Do you understand the impact of this?

With this line, we add the whole ethers lib as a dependency to our lib. This is very likely wayyyy to much things. We want to keep our lib light. At the very least, install "sub libs" - you will notice we didn't install the whole web3js lib but only subparts of it.

I recommend to stick to web3js and not add both web3js and ethers.
The impact is big. Be careful when adding libs to this project. This is not a nodejs backend project where we don't care about the size.

https://bundlephobia.com/package/ethers@6.3.0
https://bundlephobia.com/package/@erc725/erc725.js@0.17.2

Reply:
Ok I understand. I will look at installing only subpackages of ethers related to the encoding only. Not the whole ether.js library.